internal/controller/driver_controller.go

```diff
@@ -921,6 +923,24 @@ func (r *driverReconcile) reconcileControllerPluginDeployment() error {
 	return err
 }
 
+func (r *driverReconcile) ControllerPluginCsiAddonsContainerPort() corev1.ContainerPort {
+
+	var port = utils.ControllerPluginCsiAddonsContainerBasePort
+
+	// the cephFS and rbd drivers need to use different ports
+	// to avoid port collisions with host network.
+	if r.isRdbDriver() {
+		port = utils.ControllerPluginCsiAddonsContainerRbdPort
+	}
+	if r.isCephFsDriver() {
+		port = utils.ControllerPluginCsiAddonsContainerCephFsPort
+
+	}
+
+	return port
+
+}
```

Comment:
```suggetsion
func (r *driverReconcile) ControllerPluginCsiAddonsContainerPort() corev1.ContainerPort {
	// The cephFS and rbd drivers need to use different ports
	// to avoid port collisions with host network.
	port = utils.ControllerPluginCsiAddonsContainerRbdPort
	if r.isCephFsDriver() {
		port = utils.ControllerPluginCsiAddonsContainerCephFsPort

	}

	return port

}
```

Reply:
The missing part was not really the comment but the removal of a redundant if branch.

PR updated accordingly and image pushed. @Madhu-1 , @rohan47